ui/css/itcss/components/network.scss

```diff
@@ -25,6 +25,10 @@
     background-color: rgba(207, 181, 240, 0.7) !important;
   }
 
+  &.consensyszkevm-test-network .menu-icon-circle div {
+    background-color: rgba(35, 79, 213, 0.7) !important;
+  }
+
```

Comment:
Is this className used anywhere? From a search it looks like it is not

![Screenshot 2023-03-02 at 6 15 30 PM](https://user-images.githubusercontent.com/8112138/222614496-5556eb7d-dd57-4dc2-a5f5-5a6194e88b06.png)